webpages/settings/components/addon-body.js

```diff
@@ -65,6 +65,11 @@ export default async function ({ template }) {
           console.log(`Loaded default values for ${this.addon._addonId}`);
         }
       },
+      copyAddonLink() {
+        navigator.clipboard.writeText(
+          `https://scratch.mit.edu/scratch-addons-extension/settings#addon-${this.addon._addonId}`
+        );
+      },
```

Comment:
We need some sort of user input feedback, so the user knows that it was successfully copied. Maybe some sort of speech bubble on top?

Reply:
A green "Copied" label to the left of the button would be easier and just as effective.

Reply:
Replacing the button with a checkmark for a short interval might be better if the window is narrow and adding a label would cause the addon name to wrap at a different point.

Reply:
I really like your idea. Can you give me icon link/svg file? Maybe something like in read.svg?
![read](https://user-images.githubusercontent.com/89837724/180601205-0fe33898-4d5e-4b2b-931a-045908574899.svg)

Reply:
Now I'm using https://icon-sets.iconify.design/charm/tick/

Reply:
Use the existing `check.svg`.